docs/README.md

```diff
@@ -20,6 +20,7 @@ about upgrading versions, breaking changes, etc
 
 - [Configuring Your App](/docs/customization/configuration.md)
 - [Creating a Theme](/docs/customization/theme.md)
+  - [Compile Assets](/docs/customization/theme.md##compile-static-assets)
```

Comment:
`##` typo here and in the `theme.md` file

Reply:
What is wrong in the theme.md file?

Reply:
Nevermind. Was a browser quirk in how it rendered link targets on hover after visiting through the link with `##`